src/progress-circle.js

```diff
@@ -72,15 +73,25 @@ class ProgressCircle extends PureComponent {
             }
         ))
 
+        const x = width / 2
+        const y = height / 2
+
+        const extraData = {
+            width,
+            height,
+            x,
```

Comment:
this doesn't align with the other chart API's where x and y are functions.

Reply:
Do you think x and y are necessary here? I’m not sure if it makes sense to include them like in other charts since they’re always just going to be half the height/width.

Reply:
Do you want me to add the extras prop to the individual sections for PieChart/ProgressCircle or add to the common props table at the top?

Reply:
I would prefer it on the respective props section (PieChart and ProgressCircle), the common props table should not state anything specific to one component

Reply:
Done